Containerfile

```
@@ -0,0 +1,29 @@
+# Builder container
+FROM --platform=linux/amd64 registry.access.redhat.com/ubi9/go-toolset AS builder
+
+ARG GCFLAGS=""
+
+WORKDIR /app
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+
+USER 0
+RUN make build
+
+# Final runtime image
+FROM --platform=linux/amd64 registry.access.redhat.com/ubi9/ubi-minimal
+
+WORKDIR /app
+
+COPY --from=builder /app/bin/dcm-placement-api /app/
```

No need to state `/app/` after defining as the `WORKDIR`. It can be:

```suggestion
COPY --from=builder /app/bin/dcm-placement-api .
```